kafka_consumer/datadog_checks/kafka_consumer/client.py

```diff
@@ -147,6 +148,9 @@ def get_consumer_offsets(self):
         for future in self._get_consumer_offset_futures(consumer_groups):
             try:
                 response_offset_info = future.result()
+            except KafkaException as e:
+                self.log.debug("Failed to read consumer offsets for %s: %s", consumer_group, e)
+            else:
```

Comment:
Same here, we can reduce the extra nesting by `continue`ing when we catch the exception and keeping the main code out of the try-except.